charts/lightdash/values.yaml

```diff
@@ -27,6 +27,7 @@ configMap:
   DBT_PROJECT_DIR: ""
   # -- Public URL of your instance including protocol e.g. https://lightdash.myorg.com
   SITE_URL: ""
+  LIGHTDASH_SECRET: 'secret'
```

Comment:
This might be dangerous because by default everybody will use the same encryption key. I think it's better to throw an error if this isn't set. What was the original behaviour?